### PR TITLE
Development courses logo/shaina

### PR DESCRIPTION
**Pull request by boyakalyani**
PR for Logo changes

### CODE REVIEW

File: learn/src/main/res/layout/item_course.xml [hunk 1]
```diff
@@ -16,8 +16,8 @@
 
         <ImageView
             android:id="@+id/ivLogo"
-            android:layout_width="@dimen/pathway_logo_size"
-            android:layout_height="@dimen/pathway_logo_size"
+            android:layout_width="48dp"
```
Comment (damanpreet97) [inline learn/src/main/res/layout/item_course.xml]: why we did not change pathway_logo_size variables value and instead hardcode it?
We could have also created a new variable if we are not supposed to change the existing variable's value

**Review by damanpreet97 — CHANGES_REQUESTED** [review]

File: learn/src/main/res/layout/item_course.xml (hunk 1 above)
Comment (Dhan-shri) [reply learn/src/main/res/layout/item_course.xml]: @shainakhatun

File: buildSrc/src/main/kotlin/Dependencies.kt [hunk 2]
```diff
@@ -170,9 +170,9 @@ object ArrowPreferences {
 }
 
 object BuildConfigVersions {
-    const val compileSdkVersion = 33
+    const val compileSdkVersion = 31
```
Comment (surajsahani) [inline buildSrc/src/main/kotlin/Dependencies.kt:173]: @shainakhatun Don't modify this section

**Review by surajsahani — CHANGES_REQUESTED** [review]
Pleaes make proper changes

File: learn/src/main/res/layout/item_course.xml (hunk 1 above)
Comment (Dhan-shri) [reply learn/src/main/res/layout/item_course.xml]: Resolved

**Pull request merged**